target/linux/ramips/mt76x8/base-files/etc/board.d/01_leds

```diff
@@ -62,6 +62,9 @@ mediatek,linkit-smart-7688)
 rakwireless,rak633)
 	ucidef_set_led_netdev "wifi_led" "wifi" "blue:wifi" "wlan0"
 	;;
+rakwireless,rak634)
```

Comment:
This is identical to the `rakwireless,rak633`, so just add it under that case, no need to duplicate